components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -327,7 +328,7 @@ function CustomLeague:_getGameModes(args, makeLink)
 	end
 
 	local gameModes = mw.text.split(args.gamemode, ',', true)
-	table.foreach(gameModes,
+	Table.foreach(gameModes,
```

Comment:
i don't see that function in that module

`table.foreach` was deprecated in lua 5.1 (as mentioned in e.g. http://lua-users.org/wiki/TableLibraryTutorial)
so i think replacing it is a good idea, but the function should first exist

Reply:
You're right, it's `iter.forEach`